airbyte-container-orchestrator/build.gradle

```diff
@@ -9,6 +9,15 @@ dependencies {
     implementation platform(libs.micronaut.bom)
     implementation libs.bundles.micronaut
 
+    // Ensure that the versions defined in deps.toml are used
+    // instead of versions from transitive dependencies
+    implementation(libs.s3) {
```

Comment:
can we add comments in all the spots where we need to force s3 due to the break for context?

Reply:
@davinchia An additional comment beyond the one listed that talks about avoiding updates from transitive dependencies?

Reply:
yes, that in this case the transitive dependencies from micronaut 3.8 and higher are incompatible with our current helm setup. We could also link to the created issue for better visibility.